### ISSUE TITLE
about the calculation of attention distribution

### ISSUE THREAD

**Issue by jinxinglu** [jongwooko/Pytorch-MiniLM#2]
nice work!
I have a question,  
why you use "F.log_softmax" in the  "teacher_attention_dist = F.log_softmax(teacher_attention_dist, dim=-1)"?
why not use "F.softmax"?

**Comment by jinxinglu** [comment]
I got the answer. when kl_loss_fct = nn.KLDivLoss(reduction="batchmean"), here the param "log_target" in this funciton  is default to False.
